frontend/src/scenes/organization/Settings/Permissions/permissionsLogic.tsx

```diff
@@ -50,6 +67,28 @@ export const permissionsLogic = kea<permissionsLogicType>([
             },
         ],
     })),
+    listeners(({ actions }) => ({
+        updatePermission: async ({ checked, role, resourceId, resourceType }) => {
+            const accessLevel = checked ? AccessLevel.WRITE : AccessLevel.READ
+            if (role.id) {
+                const updatedRole = await api.roles.update(role.id, {
+                    [ResourceAccessLevelMapping[resourceType]]: accessLevel,
+                })
+                if (updatedRole) {
+                    actions.updateRole(updatedRole)
+                    lemonToast.success(`${role.name} role ${resourceType} edit access updated`)
```

Comment:
Opted for a toast display here indicating a successful edit access update instead of a save button since updating these organizational settings is instantaneous (toast) vs in a form (save btn). This is similar to how we have `X member has been invited` or `Your configuration has been saved` toasts in our current organization settings page when we invite a member or edit an organization setting.

Reply:
yepp